templates/minio.env.j2

```diff
@@ -18,4 +18,12 @@ MINIO_ACCESS_KEY="{{ minio_access_key }}"
 MINIO_SECRET_KEY="{{ minio_secret_key }}"
 {% endif %}
 
```

Comment:
How about we squish this up, so we don't add extra newlines when these settings aren't used.